### ISSUE TITLE
Chaos Waste: Charon's Keep

### ISSUE THREAD

**Issue by Stinkyweed** [WarEmu/WarBugs#8934]
There is a terrain glitch on the NE corner of the keep that allows you to bypass the outer door; where the wall of the keep tower and the rock hill meet.  It was there on live and is still there.

**Comment by Stophy22** [comment]
Some advice, maybe put a screenshot in for them to see visually what you're talking about. But nice find 👍

**Comment by Stinkyweed** [comment]
Sorry Chris, I'll grab one and add it to the post tonight after work.

Sent from Yahoo Mail on Android 
 
  On Tue, Jan 17, 2017 at 3:57 AM, Christopher Savary<notifications@github.com> wrote:   
Some advice, maybe put a screenshot in for them to see visually what you're talking about. But nice find 👍

**Comment by Xanics** [comment]
Its a bit difficult, but the OP is correct, there is a spot. At the corner. when you try hard enough, its possible to get up.

![image](https://cloud.githubusercontent.com/assets/14800906/22142150/69e6f05c-def5-11e6-8a46-ffbfb088f849.png)

![xanics_018](https://cloud.githubusercontent.com/assets/14800906/22142160/7310c6a8-def5-11e6-8c88-db3d13ae0216.jpg)

![xanics_021](https://cloud.githubusercontent.com/assets/14800906/22142226/b8b14e76-def5-11e6-97a9-99f66bc21ed6.jpg)

![xanics_024](https://cloud.githubusercontent.com/assets/14800906/22142258/da9cfe40-def5-11e6-85e2-71f78ab2fa41.jpg)

![xanics_011](https://cloud.githubusercontent.com/assets/14800906/22142280/f0b029dc-def5-11e6-92ef-069cf3a17c38.jpg)

**Comment by Luth-RoR** [comment]
Thanks.

**Comment by Blaqwar** [comment]
I actually tested that spot for about 15 min in the past, I couldn't get there with an elf. Not sure if it's possible but if it is it's hellishly difficult.

**Comment by Stinkyweed** [comment]
I tested a few moments ago and was still able to get up.  The exact spot is correcrly identified in the previous email images.  Once you get into/under the terrain it pops you up to the top.  Go into the hill, rather than up it.

Sent from Yahoo Mail on Android 
 
  On Fri, Jan 20, 2017 at 1:30 PM, Blaqwar<notifications@github.com> wrote:   
I actually tested that spot for about 15 min in the past, I couldn't get there with an elf. Not sure if it's possible but if it is it's hellishly difficult.
